crates/air-lib/trace-handler/src/data_keeper/trace_slider.rs

```diff
@@ -83,6 +83,7 @@ impl TraceSlider {
     }
 
     pub(crate) fn set_subtrace_len(&mut self, subtrace_len: usize) -> KeeperResult<()> {
+        println!("set_subtrace_len subtrace_len: {}", subtrace_len);
```

Comment:
don't forget to delete it before merging

Reply:
Those lines accedently slipped into the patch. Thx for your vigilance.